doc/chapters/api/index.rst

```diff
@@ -32,102 +32,187 @@ BindMount
 Binds a path on the host to the container. Any missing parent directories in
 the path will be created.
 
-:Parameters:
-        :containerID: ``int32`` The ID obtained by CreateContainer method.
-        :pathInHost: ``string`` absolute path in the host.
-        :pathInContainer: ``string`` the absolute path to mount to in the container.
-        :readOnly: ``bool`` indicates whether the directory is read-only or not.
-
-|
+Parameters
+##########
+* containerID: ``int32`` The ID obtained by CreateContainer method.
+* pathInHost: ``string`` absolute path in the host.
+* pathInContainer: ``string`` the absolute path to mount to in the container.
+* readOnly: ``bool`` indicates whether the directory is read-only or not.
+
+Error sources
```

Comment:
And yes, these items are connected, although loosely, to the error message one gets. It is intentionally not closer connected that this, since that could lead to a maintenance problem.

Moreover, I think this list is more exhaustive than the error messages one can get, since some errors are wrapped in others etc. I think it makes sense to keep them in just to give the user a hint of what might go wrong.